src/services/apiCore.ts

```diff
@@ -1,5 +1,5 @@
 import axios from 'axios';
 
 export const axiosInstance = axios.create({
-  baseURL: process.env.API_BASE_URL,
+  baseURL: process.env.NEXT_PUBLIC_API_BASE_URL,
```

Comment:
Environment variables in nextjs must start with `NEXT_PUBLIC_` to be accessible on the browser side.